Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -441,7 +525,7 @@ public override void Subscribe(LiveNodePacket job, IEnumerable<Symbol> symbols)
             WebSocket.Send(json);
 
 
-            OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Information, -1, "GDAXBrokerage.Subscribe: Sent subcribe."));
+            OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Information, -1, "GDAXBrokerage.Subscribe: Sent subscribe."));
```

Comment:
Instead of -1, perhaps we could use `"Subscribe"` or `"SentSubscribe"` as the code

Reply:
Only fixed the typo on that line, changing codes should be also done in other places, next PR :)